### ISSUE TITLE
numerical tags causes serious problem!

### ISSUE THREAD

**Issue by nazarifard** [owncloud/ocis#9610]
if user add number as a tag to one folder, the folder will be missed entirely!
for example; adding 1234 as a tag will cause the resource disappear totally.

**Comment by phil-davis** [comment]
@PrajwolAmatya do we have some test coverage for tags, and the case where a tag is numeric?

**Comment by nazarifard** [comment]
> @PrajwolAmatya do we have some test coverage for tags, and the case where a tag is numeric?

Yes, just assign a 1234 as a tag to a folder and press enter. I can't show the result because OCIS doesn't work after this case and can't show the parent folder that contains any resource with numeric tags!

**Comment by nazarifard** [comment]
![image](https://github.com/user-attachments/assets/9d368da0-78c8-4ce1-8afe-19ec5d7ed539)

after refresh the page, OCIS can't find the resource.

![image](https://github.com/user-attachments/assets/f2abad17-694c-4050-804f-4dd4a388d23d)

**Comment by PrajwolAmatya** [comment]
> @PrajwolAmatya do we have some test coverage for tags, and the case where a tag is numeric?

I can reproduce this, adding numeric tag and refreshing the page the oCIS stops working.
We do have test cases for tags but not specifically with the numerical tags. I will create a separate issue to add tests for adding numeric tags to a resource.

**Comment by kobergj** [comment]
Awesome. You loose your complete personal space if you add numerical tags to a resource. Works also in project spaces. Needs a fix asap.
